code/InitializeQuiz.js

```diff
@@ -1,5 +1,5 @@
-const { initializeQuiz } = require("./lib/quizLogic.js")
+import quizLogic from "./lib/quizLogic.js";
 
-exports.function = function() {
-  return initializeQuiz()
+export default function () {
+  return quizLogic.initializeQuiz();
```

Comment:
in quizLogic.js instead of
```
export default {
  initializeQuiz,
  updateQuiz,
};
```
if you export functions separately you'll be able to keep the initial implementation which looks better.